src/middleware/__tests__/batch.test.js

```diff
@@ -82,14 +120,14 @@ describe('batchMiddleware', () => {
       matcher: '/graphql/batch',
       response: {
         status: 200,
-        body: [{ data: {} }, { id: 2, data: { ok: 2 } }],
+        body: [{ id: 2, data: { ok: 2 } }],
       },
       method: 'POST',
     });
 
     const req1 = mockReq(1);
     const req2 = mockReq(2);
-    const req3 = mockReq(3);
```

Comment:
The test description says duplicated ids, but the IDs were not actually duplicated :)